controllers/index.js

```diff
@@ -50,3 +51,18 @@ exports.postCommentByReviewId = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.patchReviewByReviewId = (req, res, next) => {
+  const {
+    params: { review_id },
+    body: { inc_value },
+  } = req;
+  Promise.all([
+    fetchReviewById(review_id),
+    updateReviewByReviewId(review_id, inc_value),
+  ])
+    .then(([_, updated_review]) => {
```

Comment:
if you do choose to leave this, I think it would be cleaner to switch the order of the promises rather than doing empty restructuring here